src/static_types.rs

```diff
@@ -99,6 +99,9 @@ pub mod westend {
 		>(16)
 	);
 
+	// SYNC https://github.com/paritytech/polkadot-sdk/blob/master/polkadot/runtime/westend/src/lib.rs#L451-#L453.
+	pub const SIGNED_PHASE_LENGTH: u64 = 150;
```

Comment:
This is a temp fix but I wonder if there are any ideas to make sure this value is up to date throughout all chains? or should we just keep a sensible upper bound since this is just used for the tx mortality?

If we leave it as is, I'd suggest us to rename `SIGNED_PHASE_LENGTH` to `SIGNED_PHASE_LENGTH_ESTIMATE` or something similar so that the const is now mistakenly used in the future.

Reply:
it's removed now and I changed it to use `babe::epoch_duration` as upper bound/tx mortality ok?